lib/util.js

```diff
@@ -146,9 +146,29 @@ function getAssertionComponents(element) {
   return { code: code.trim(), assertion: assertion.trim(), expected: expected ? expected.trim() : expected };
 }
 
-async function executeTests() {
-  const { stdout } = await execa('node', [`node_modules/.bin/qunit`, `test.js`]);
-  console.log(stdout);
+async function executeTests(inspect) {
+  let args = [];
+  if (inspect) {
+    let inspectBrk = `--inspect-brk`;
+    if (/\d/.test(inspect)) {
+      inspectBrk += `=${inspect}`;
+    }
+    args.push(inspectBrk);
+  }
+  args.push.apply(args, [`node_modules/.bin/qunit`, `test.js`]);
+  return new Promise ((res, rej) => {
+    const child_proc = execa('node', args);
```

Comment:
do we need to wrap this in a promise, execa already does return a promise.

Reply:
We need to wrap it incase of `--inspect-brk` otherwise it doesn't print anything and just hangs in there unless we open the dev tool and continue the execution, it doesn't prompt to open dev tools.